apps/src/levelbuilder/course-editor/CourseEditor.js

```diff
@@ -416,7 +416,8 @@ class CourseEditor extends Component {
             <div>
               The dropdown(s) below represent the ordered set of units in this
               course. To remove a unit, just set the dropdown to the default
-              (first) value.
+              (first) value. Please referesh the page after every edit to get
+              the most uptodate list of units.
```

Comment:
I was mainly thinking of a scenario where we want to reorder units. In that case, one of the units that is currently in the unit group would be removed, but this won't be available in the other drop downs. I'm thinking there can be a way to address this by adding all the units in current course/unit group to the list of options. I'll explore this a bit more.

Reply:
> I'm thinking there can be a way to address this by adding all the units in current course/unit group to the list of options.

Yes, this sounds promising!

The other test case which comes to mind is, if I have a list of unit in a course:
csp1
csp2
csp3

what happens if I select csp1 and change it to csp3. I think you would want it to end up like this:
csp3
csp2
[blank]

Reply:
Yeah, I was thinking of that as well. I am hoping there is some backend validation that will block have the same unit twice in a unit group. I'd guess the client side behavior today would be that the lists will show up as

csp3
csp2
csp3

and I'm thinking as long as backend blocks it, this should be still ok. I'll test this locally to confirm.

Reply:
I agree, it could be ok for the client to be in a "bad" state temporarily as long as we have validations on the backend to keep us from getting into a bad state.